pallets/gear/src/runtime_api.rs

```diff
@@ -305,10 +305,10 @@ where
     }
 
     fn code_with_memory(program_id: ProgramId) -> Result<CodeWithMemoryData, String> {
-        let (program, _bn) = ProgramStorageOf::<T>::get_program(program_id)
+        let item = ProgramStorageOf::<T>::get_program(program_id)
```

Comment:
I don't really like "item" naming across the code

Reply:
What about `program_item` or just simply `program`?

Reply:
"program" is okay
any other things can be obtained from struct declaration